src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,236 @@
+
+Imported changes and notes
+==========================
+
+Upgrade to Bitcoin Core 0.10.3
+------------------------------
+
+The underlying base of Omni Core was upgraded to Bitcoin Core 0.10.3.
+
+Please see the following release notes for further details:
+
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.0.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.1.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.2.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.3.md
+
+Headers-first synchronization
+-----------------------------
+
+With the upgrade to Bitcoin Core 0.10 headers-first synchronization is supported and blocks are downloaded in parallel.
+
+Blocks are no longer stored in order on disk, and as result, the block files and databases are not backwards-compatible with older versions of Bitcoin Core prior 0.10.
+
+Dust threshold values
+---------------------
+
+The default `"minrelaytxfee"` was raised from `0.00001 BTC` to `0.00005 BTC`in Bitcoin Core 0.10.3 as temporary measure against massive memory pool bloat.
+
+This has an impact on the output values of Omni transactions, which are chosen to be as low as possible. As per default, Omni transactions created with Master Core had output values between `0.00000546 BTC` and `0.00000882 BTC`, whereby the new output values are between `0.0000273 BTC` and `0.0000441 BTC`.
+
+To continue to create transactions with lower values, start Omni Core with `-minrelaytxfee=0.00001` or add the following to your `bitcoin.conf`:
+```
+minrelaytxfee=0.00001
+```
+
+A lower `"minrelaytxfee"` may result in higher memory consumption, but is usually safe to use.
+
+Rebranding to Omni Core
+-----------------------
+
+Master Core was re-branded to Omni Core on all levels:
+
+- `mastercored`, `mastercore-cli` and `mastercore-qt` were renamed to `omnicored`, `omnicore-cli` and `omnicore-qt`
+- the debug log file `mastercore.log` was renamed  to `omnicore.log`
+- the hardcoded token SP#1 was renamed from `"MasterCoin"` to `"Omni"`
+- the hardcoded token SP#2 was renamed from `"Test MasterCoin"` to `"Test Omni"`
+- the user interface now refers to `"Omni"`, `"OMNI"`, `"Test Omni"` or `"TOMNI"` instead of `"Mastercoin"`, `"MSC"`, `"Test Mastercoin"` or `"TMSC"`
+
+Incompatible API changes
+------------------------
+
+The field `"subaction"` in the RPC response for traditional DEx orders of `"gettransaction_MP"` as well as `"listtransaction_MP"` was renamed to `"action"`.
+
+The values associated with `"action"` were renamed from `"New"`, `"Update"`, `"Cancel"` to `"new"`, `"update"`, `"cancel"`.
+
+Notable changes
+===============
+
+Feature activations
+-------------------
+
+TODO
+
+Data embedding with OP_RETURN
+-----------------------------
```

Comment:
How about this:

**Data embedding with OP_RETURN**

Omni Core 0.0.10 contains support for Class C transactions which move the data encoding from multisig data outputs to an "OP_RETURN" output.  "OP_RETURN" outputs carry no output value and can be immediately pruned.

In addition to migrating data encoding to OP_RETURN, the requirement to send an output to the Exodus address has also been removed with Class C.  Instead the bytes ```6f6d6e69``` ("omni") are prefixed to the data payload to provide marker identification.

This helps to address a common criticism of systems that store data on the Bitcoin blockchain; "UTXO bloat".  Since our Class C transactions no longer store our data in spendable outputs, there is no need to store them in the UTXO set.  Since the outputs are not stored in the UTXO set, our new Class C transactions do not contribute to its growth.

Due to size and count restrictions with OP_RETURN outputs currently enforced by the Bitcoin network, the client will automatically fall back to Class B (multisig) in the event that a transaction is too large to send via Class C (for example a property creation with lots of metadata).

Note: Whilst OmniCore 0.0.10 contains support for Class C transactions, they are not yet allowed by the protocol.  Class C transactions will be activated via the feature activation system shortly after Omni Core 0.0.10 is released and we are confident the user base has upgraded.